nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/NviService.java

```diff
@@ -29,6 +31,12 @@ public void upsertCandidate(CandidateEvaluatedMessage evaluatedCandidate) {
         }
     }
 
+    //TODO: Remove JacocoGenerated when other if/else cases are implemented
+    @JacocoGenerated
+    public Optional<Candidate> getCandidate(URI publicationIdentifier) {
```

Comment:
Do we want to return Optional here? Maybe return Candidate and throw not found if does not exist?